index.js

```diff
@@ -30,20 +32,34 @@ module.exports = function(options) {
   debug('filename: ', filename);
 
   if (typeof config === 'string') {
-    configPath = config;
+    configPath = path.dirname(config);
     config = module.exports._readConfig(config);
     debug('converting given config file ' + configPath + ' to an object:\n', config);
   }
 
+  if (configPath && !fs.statSync(configPath).isDirectory()) {
```

Comment:
This is needed for the existing tests to pass since configPath is set to a file and not a directory.